mapping/requirements_officejs.json

```
@@ -5088,6 +5110,17 @@
 							}],
 							"availability": "GA"
 						},
+						{
+							"name": "WordApiDesktop",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.0.17628.20114",
```

Seems like this should be 17928 based on https://learn.microsoft.com/en-us/officeupdates/update-history-microsoft365-apps-by-date.

```suggestion
									"build": "16.0.17928.20114",
```